### ISSUE TITLE
polysemy-plugin-0.2.0.1 test suite build failure with polysemy-0.3.0.0

### ISSUE THREAD

**Issue by DanBurton** [isovector/polysemy#93]
As seen on the Stackage build server:

```
test/InlineRecursiveCallsSpec.hs:49:16: error:
    • The constructor ‘Yo’ should have 5 arguments, but has been given 4
...
test/InlineRecursiveCallsSpec.hs:66:16: error:
    • The constructor ‘Yo’ should have 5 arguments, but has been given 4

```

**Comment by isovector** [comment]
Sorry about this! It's a breaking change, but in an internal module. This is an acceptable test failure I think.

**Comment by isovector** [comment]
Fixed in `polysemy-plugin-0.2.0.2`
